dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseOne.java

```diff
@@ -3754,4 +3764,20 @@ public Node visitOther(Tree tree, Void p) {
     private TypeMirror getTypeMirror(Class<?> clazz) {
         return TypesUtils.typeFromClass(clazz, types, elements);
     }
+
+    /**
+     * Returns the TypeMirror for the given class, or {@code null}.
+     *
+     * @param clazz a class
+     * @return the TypeMirror for the class
+     */
+    private @Nullable TypeMirror maybeGetTypeMirror(Class<?> clazz) {
+        String name = clazz.getCanonicalName();
+        assert name != null : "@AssumeAssertion(nullness): assumption";
```

Comment:
The method only works for classes with a canonical name.  That is worth mentioning in its Javadoc, and then here "assumption" can be more specific.

Reply:
Done